lib/ansible/modules/cloud/ovh/ovh_dns.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# ovh_dns, an Ansible module for managing OVH DNS records
+# Copyright (C) 2014, Carlos Izquierdo <gheesh@gheesh.org>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+
+DOCUMENTATION = '''
+---
+module: ovh_dns
+author: Albin Kerouanton @NiR-
+short_description: Manage OVH DNS records
+description:
+    - Manage OVH (French European hosting provider) DNS records
+version_added: "2.3"
+notes:
+    - Uses the python OVH Api U(https://github.com/ovh/python-ovh).
+      You have to create an application (a key and secret) with a consummer
+      key as described into U(https://eu.api.ovh.com/g934.first_step_with_api)
+requirements: [ "ovh" ]
+options:
+    domain:
+        required: true
+        description:
+            - Name of the domain zone
+    name:
```

Comment:
The "name" is not very clear, is it subdomain e.g; db1 means we change the record db1.mydomain.com ?
I tried the api but was unable to change the main A record mydomain.com, could you explain suitable parameter ?